frontend/src/components/CustomDropdown/CustomDropdown.js

```diff
@@ -0,0 +1,195 @@
+import React from "react";
+// nodejs library that concatenates classes
+import classNames from "classnames";
+// nodejs library to set properties for components
+import PropTypes from "prop-types";
+
+// @material-ui/core components
+import { makeStyles } from "@material-ui/core/styles";
+import MenuItem from "@material-ui/core/MenuItem";
+import MenuList from "@material-ui/core/MenuList";
+import ClickAwayListener from "@material-ui/core/ClickAwayListener";
+import Paper from "@material-ui/core/Paper";
+import Grow from "@material-ui/core/Grow";
+import Divider from "@material-ui/core/Divider";
+import Icon from "@material-ui/core/Icon";
+import Popper from "@material-ui/core/Popper";
+
+// core components
+import Button from '../CustomButtons/Button'
+
+import styles from '../../assets/jss/material-kit-react/components/customDropdownStyle';
+
+const useStyles = makeStyles(styles);
+
+export default function CustomDropdown(props) {
```

Comment:
chuyển sang `type Props = {}` thay thế cho props-types
 ```suggestion
export const CustomDropdown: React.FC<Props> = (props) => {
```